### PR TITLE
contributing.md introduced

### PR DESCRIPTION
**Pull request by jinek**
Tried to keep the description as simple as possible.
Things, like code style/formatting are defined by DotSettings and editorconfig. Otherwise resharper is supposed to be used for formatting locally or on CI.

### CODE REVIEW

File: contributing.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
+# Contributing Guide
+
+## Preparation
+To compile and debug the application it's necessary to install [dotnet 5 sdk](https://dotnet.microsoft.com/en-us/download/dotnet/5.0). It's possible to use any compliant tools for coding and debugging.
```
Comment (vabka) [inline contributing.md:4]: Are there any plans for .NET 6?

File: contributing.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+
+## Tickets And Discussions
+Feel free to open a [ticket](https://github.com/jinek/Consolonia/issues) or a [discussion](https://github.com/jinek/Consolonia/discussions) in case you have issues, need help, or just have some ideas.
+
+In case of a bug, first try to check whether issue exist in [AvaloniaUI](https://github.com/AvaloniaUI/Avalonia) - Avalonia community also accepts tickets, we should consider fixing it in Avalonia rather than in Consolonia if applicable.
+
+## Code Style
+Projects are setup to utilize [dotnet analyzers](https://docs.microsoft.com/en-us/dotnet/framework/code-analyzers).
+All warning are considered as errors.
+Formatting and styling is set by editorconfig.yml which can be recognized by [JetBrains Resharper or Rider](https://www.jetbrains.com/resharper/). If you don't have one, just make a pull request and check build statuses. It does automatic formatting and checks the style.
```
Comment (vabka) [inline contributing.md:14]: editorconfig also recognizable by [dotnet format](https://github.com/dotnet/format) - it's a great tool if you do not have Rider or something went wrong with IDE.

Also I need we need to consider about adding dotnet format to git/hooks

File: contributing.md (hunk 1 above)
Comment (jinek) [reply contributing.md:4]: Not planned yet, but should be simple to implement

File: contributing.md (hunk 2 above)
Comment (jinek) [reply contributing.md:14]: I have tried to format with "dotnet format", it works differently from Resharper. Either config needs to be adjusted or "dotnet format" itself fixed.

Comment (vabka) [reply contributing.md:14]: Resharper has its own properties in editorconfig.

I think it can be fixed in another PR

**Comment by vabka** [comment]
Lgtm

**Pull request merged**